Framework7/src/data/account/account.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../base_class_for_data_classes'
+
+class Account < BaseClassForDataClasses
+  attr_accessor :id,
+                :name,
+                :plan,
+                :status,
+                :created_at,
+                :updated_at,
+                :kind
+
+  @hash=Hash.new
+  def initialize(values = {})
+    super(values)
+
+  end
+  def to_hash
```

Comment:
Maybe this method could be moved to the BaseClassForDataClasses, with this all your team mates could use it